lib/exe/css.php

```diff
@@ -344,6 +360,37 @@ function css_fixreplacementurls($replacements, $location) {
 }
 
 /**
+ * Wrapper for the files, content and mediatype for the event CSS_STYLES_INCLUDED
+ *
+ * @param string $mediatype type ofthe current media files/content set
+ * @param array $files set of files that define the current mediatype
+ * @return array
+ */
+function css_filewrapper($mediatype, $files=array()){
+    return array(
+            'files'                 => $files,
+	        'mediatype'             => $mediatype,
+	        'encapsulate'           => in_array($mediatype, array('screen', 'print')),
```

Comment:
As per #982 - what if the style.ini definition does not define the mediatype but more like a context type that authors could reference and then change into a whitelisted mediatype? I do not really like to use naming conventions for this. It makes the system a bit static again. Also it would not really allow re-using predefined sets for different contexts (#1003)